pkgs/top-level/all-packages.nix

```
@@ -20418,13 +20418,17 @@ with pkgs;
     autoreconfHook = buildPackages.autoreconfHook269;
   };
 
-  SDL = callPackage ../development/libraries/SDL ({
+  SDL_classic = callPackage ../development/libraries/SDL ({
```

```suggestion
  SDL_1 = callPackage ../development/libraries/SDL ({
```
or is classic a widely used name?